pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -563,16 +558,16 @@ private void doAuthentication(PGStream pgStream, String host, String user, Prope
                   if (LOGGER.isLoggable(Level.FINE)) {
                     LOGGER.log(Level.FINE, "Using SSPI: {0}, gsslib={1} and SSPI support detected", new Object[]{useSSPI, gsslib});
                   }
-                }
 
-                if (useSSPI) {
-                  /* SSPI requested and detected as available */
-                  sspiClient.startSSPI();
-                } else {
-                  /* Use JGSS's GSSAPI for this request */
-                  org.postgresql.gss.MakeGSS.authenticate(pgStream, host, user, password,
-                      PGProperty.JAAS_APPLICATION_NAME.get(info),
-                      PGProperty.KERBEROS_SERVER_NAME.get(info), usespnego);
+                  if (useSSPI) {
```

Comment:
This change looks suspicious, and I would refrain from doing that considering SSPI is not covered by the CI tests.

Would you please revert the change?

Reply:
reverted